pages/linux/create_ap.md

```diff
@@ -0,0 +1,35 @@
+# create_ap
+
+> Create an AP (Access Point) at any channel.
+
+- No passphrase (open network):
```

Comment:
In my opinion the command descriptions should start with verbs as it gets easier to understand exactly what does the command do. 

This comment applies to all other command descriptions (except the last one).

Reply:
What I @schneiderl means is that rather than this:

```
No passphrase
```

....we prefer this:

```
Create an access point without a passphrase
```